conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class FrugallyDeepConan(ConanFile):
+    name = "frugally-deep"
+    version = "0.1-p0"
```

Comment:
Delete version variable as it will be set dynamically in Travis/Appveyor every time you release with a new tag